specification/vmware/resource-manager/Microsoft.AVS/stable/2023-09-01/vmware.json

```diff
@@ -10304,7 +10304,7 @@
         },
         "leaseTime": {
           "type": "integer",
-          "format": "int32",
+          "format": "int64",
           "description": "DHCP Server Lease Time."
         }
```

Comment:
This is the fix for incorrect TypeSpec
https://github.com/Azure/autorest.java/issues/2803#issuecomment-2151352378

It was int64 in 2023-03-01 stable
https://github.com/Azure/azure-rest-api-specs/blob/3f47536facebaad1b039b3711965debc9c53ded9/specification/vmware/resource-manager/Microsoft.AVS/stable/2023-03-01/vmware.json#L6923-L6927